staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2579,6 +2615,20 @@ type PodCondition struct {
 	Message string `json:"message,omitempty" protobuf:"bytes,6,opt,name=message"`
 }
 
+// ResourcesResizeStatus shows status of resources resize of a pod's containers.
+type ResourcesResizeStatus string
+
+const (
+	// Pod resources resize has been requested and will be evaluated by node.
+	Proposed ResourcesResizeStatus = "Proposed"
```

Comment:
when defining a const symbol, it needs to be something like "PodResizeStatusProposed"

Reply:
fixed.

Reply:
fixed in commit https://github.com/kubernetes/kubernetes/pull/102884/commits/b60c5d045bb85eacae064e209994c4afe22a4e5f